samples/cpp/gstreamer_pipeline.cpp

```diff
@@ -30,32 +29,36 @@ class GStreamerPipeline
         if (cmd_parser->has("help"))
         {
             cmd_parser->printMessage();
-            exit_code = -1;
+            throw 1;
```

Comment:
In case of "-help" we can just call `exit(0);` after the message output.
For other errors: `throw std::exception("Error message here");`

Exception handling in main:
```
catch (const std::exception& e)
{
    std::cerr << "Exception: " << e.what() << std::endl;
    return 1;
}
```

Reply:
@alalek replaced